pkg/util/deployment/deployment.go

```diff
@@ -270,6 +279,16 @@ func waitForReplicaSetUpdated(c clientset.Interface, desiredGeneration int64, na
 	})
 }
 
+func waitForPodsHashPopulated(c clientset.Interface, namespace, name string) error {
```

Comment:
Hmm. Nevermind about moving to pkg/client/unversioned/conditions.go. Looks like the types are different. 

I'm fine with leaving this here for now.